kolibri/core/webpack/hooks.py

```diff
@@ -158,11 +159,7 @@ def build_path(self):
         An auto-generated path to where the build-time files are stored,
         containing information about the built bundles.
         """
-        return os.path.join(
-            os.path.abspath(os.path.dirname(__name__)),
-            self._module_file_path,
-            "build"
-        )
+        return resource_filename('kolibri.core', 'build')
```

Comment:
Yup! Although I think it might require people to do a `pip install -e .` now for their dev environments to work, since python isn't aware of the package yet before you do so.